providers/circleci.js

```diff
@@ -9,14 +9,15 @@ class CircleCi extends BaseProvider {
     }
 
     async parseData(req) {
-        this.payload.setEmbedColor(0x000000);
+        this.payload.setEmbedColor(0x343433);
         this.payload.addEmbed({
-            title: "Build #" + this.body.payload.build_num,
+            title: `Build #${this.body.payload.build_num}`,
             url: this.body.payload.build_url,
             author: {
-                name: this.body.payload.reponame
+                name: `${this.body.payload.reponame}:${this.body.payload.branch}`,
+                icon_url: `https://github.com/${this.body.payload.committer_name}.png`
             },
-            description: "**Outcome**: " + this.body.payload.outcome
+            description: `[\`${this.body.payload.vcs_revision.slice(0,7)}\`](${this.body.payload.compare}) : ${this.body.payload.subject} - ${this.body.payload.committer_name}\n\`Outcome\`: ${this.body.payload.outcome}`
```

Comment:
Would it be benefitial to add a length limiter to the subject line? I'm not sure how long these could get, but it might be a good idea to set a maximum length, as seen in the screenshots you've posted from the github payloads. Here is an example of what I mean: https://github.com/Commit451/skyhook/blob/master/providers/bitbucket.js#L61

Reply:
Resolved in 91cb5ff